server/lib/nl/common/constants.py

```diff
@@ -421,6 +437,14 @@
         "RCP 4.5 is more likely than not to result in global temperature rise between 2 °C and 3 °C by 2100.",
     "ProjectedMin_Until_2050_DifferenceRelativeToBaseDate1981To2010_Min_Temperature_RCP60":
         "RCP 6.0 simulates conditions through 2100 making the global temperature rise between 3 °C and 4 °C by 2100.",
+    "DiffRelativeToAvg_1980_2010_MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP245": 
```

Comment:
These are definitions of RCP,  maybe we should update these to include the SSP too?

Reply:
Done